src/test/java/com/wikia/webdriver/common/core/imageutilities/ImageComparison.java

```diff
@@ -85,6 +85,7 @@ public boolean areImagesDifferent(BufferedImage image1, BufferedImage image2, in
     for (int x = 0; x < image1.getWidth(); x++) {
       for (int y = 0; y < image1.getHeight(); y++) {
         if (image1.getRGB(x, y) == image2.getRGB(x, y)) {
+          System.out.println();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS106)